<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmLogIn</class>
 <widget class="QWidget" name="FrmLogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color: rgba(0, 170, 255,100);}
#frame{border: 2px solid rgba(100,100,100,100);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>69</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrEdit">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>60</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logIn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SignIn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="btn_savePassword">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
